client/layout/guided-tours/tours/checklist-site-title-tour/index.js

```diff
@@ -28,10 +27,13 @@ export const ChecklistSiteTitleTour = makeTour(
 				<Fragment>
 					<p>
 						{ translate(
-							'Update the {{siteTitleButton/}} field with a descriptive name ' +
-								'to let your visitors know which site they’re visiting.',
+							'Update the {{siteTitleButton/}} and {{SiteTaglineButton/}} fields ' +
+								'to let visitors clearly identify your site.',
 							{
-								components: { siteTitleButton: <SiteTitleButton /> },
+								components: {
+									siteTitleButton: <SiteTitleButton />,
+									siteTaglineButton: <SiteTaglineButton />,
```

Comment:
Thanks for renaming the component, but did you have a chance to test it? It looks broken now:

![](https://cldup.com/BhHqREKxTa.png)

Reply:
Its fixed now, the rename I caused this, I should I have caught this. will test more rigorously.